examples/preact/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "example-preact",
+  "private": true,
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "storybook": "start-storybook --port 6006",
+    "build-storybook": "build-storybook",
+    "preview-storybook": "http-server storybook-static --port 6006 --silent",
```

Comment:
This should be a different port from all the other examples, and the `storybook` script above and `test` below should be changed as well, to the same new, unique port.  This lets us build/test all of the examples at the same time in our CI.